library/src/main/java/com/novoda/downloadmanager/lib/NotificationVisibility.java

```diff
@@ -0,0 +1,35 @@
+package com.novoda.downloadmanager.lib;
+
+import android.support.annotation.IntDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+public class NotificationVisibility {
+    /**
+     * This download is visible but only shows in the notifications
+     * while it's in progress.
+     */
+    public static final int ONLY_WHEN_ACTIVE = 0;
```

Comment:
Eventually it will be, but just to keep this PR not so massive we've chosen not to include it now.